spec/spec_helper.rb

```diff
@@ -0,0 +1,16 @@
+ENV['RACK_ENV'] ||= 'test'
+
+require 'rubygems'
+require File.expand_path('../../environment', __FILE__)
+
+require 'rack/test'
+
+RSpec.configure do |config|
+    config.include Rack::Test::Methods
```

Comment:
Make indentation consistent. You use two spaces everywhere else, so do so in this file as well.